apps/api/src/app/events/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -544,17 +556,17 @@ export class TriggerEvent {
             },
           })
         );
-      });
+      }
     } else {
       await this.messageRepository.updateMessageStatus(
         message._id,
         'warning',
         null,
         'mail_unexpected_error',
-        'Subscriber does not have an email address'
+        'Subscriber does not have an email address or active integration'
```

Comment:
Can we create a better separation for this error? I would prefer to have 2 separate errors: 
- Susbscriber don't have phone
- No active integration was found